R/polynomial-dissimilarity.R

```diff
@@ -1,5 +1,7 @@
+library(pracma)
```

Comment:
Remove the library line.  When using an exported function there are a couple of ways we need to import it. This walks you through how to do it with [roxygen commens](https://laderast.github.io/2019/02/12/package-building-description-namespace/) . You can also just explicitly reference the namespace like `pracma::dot` which is what I usually do.

Once that works you need to explicitly add the import to the `DESCRIPTION` file.  Here's a way to do that automatically with the `usethis` package.

```R
usethis::use_package("pracma")
```

Reply:
How do I check the output?

Reply:
Click details where it says `Some checks were not successful`.  Alternatively, you can click the check button in RStudio  under the build package to run package checks.  Or you can use:

```R
devtools::check(document=FALSE)
```